src/RuleSet/DeclarationBlock.php

```diff
@@ -116,16 +116,16 @@ public function setSelectors($mSelector, $list = null): void
     /**
      * Remove one of the selectors of the block.
      *
-     * @param Selector|string $mSelector
+     * @param Selector|string $selectorToRemove
      */
-    public function removeSelector($mSelector): bool
+    public function removeSelector($selectorToRemove): bool
     {
-        if ($mSelector instanceof Selector) {
-            $mSelector = $mSelector->getSelector();
+        if ($selectorToRemove instanceof Selector) {
+            $selectorToRemove = $selectorToRemove->getSelector();
         }
-        foreach ($this->aSelectors as $key => $selector) {
-            if ($selector->getSelector() === $mSelector) {
-                unset($this->aSelectors[$key]);
+        foreach ($this->selectors as $key => $subSelector) {
+            if ($subSelector->getSelector() === $selectorToRemove) {
```

Comment:
I think `$selector` singular was fine here for the loop variable.  `$subSelector` is a strange name.  (The parameter name change is good.)